travelwise/frontend/views.py

```diff
@@ -54,10 +53,8 @@ def login_view(request):
             if user is not None:
                 login(request, user)
                 return redirect("home")
-        else:
-            messages.error(request, 'Invalid username or password')
-
-    form = AuthenticationForm()
+    else:
+        form = AuthenticationForm()
```

Comment:
Probably should be a 405 error.

Reply:
I want it to also work for a get request, when they are just viewing the page.

Reply:
In addition to supporting errors if the form is not valid.

Reply:
I see, you use that for serving the page itself as well, that's fine.